### PR TITLE
Adds helpers to avoid << use

### PR DESCRIPTION
**Pull request by Cyberboss**
SEND_IMAGE, SEND_SOUND, SEND_TEXT, WRITE_FILE

Also fixed some things that should have been sound_to_playing_players and to_chat

### CODE REVIEW

File: code/__HELPERS/_logging.dm [hunk 1]
```diff
@@ -1,3 +1,10 @@
+//wrapper macros for easier grepping
+#define DIRECT_OUTPUT(A, B) A, B
```
Comment (MrStonedOne) [inline code/__HELPERS/_logging.dm]: This doesn't seem right.

**Comment by optimumtact** [comment]
didn't I already do this

**Comment by optimumtact** [comment]
oh I see you moved it

**Comment by lzimann** [comment]
Keep the `<<` usage to the minimum as possible, pretty good.

File: code/modules/client/preferences_savefile.dm [hunk 2]
```diff
@@ -193,34 +193,34 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		return 0
 	S.cd = "/"
 
-	S["version"] << SAVEFILE_VERSION_MAX		//updates (or failing that the sanity checks) will ensure data is not invalid at load. Assume up-to-date
+	WRITE_FILE(S["version"] , SAVEFILE_VERSION_MAX		//updates (or failing that the sanity checks) will ensure data is not invalid at load. Assume up-to-date)
```
Comment (lzimann) [inline code/modules/client/preferences_savefile.dm]: parenthesis here is in the end of the comment

File: code/modules/client/preferences_savefile.dm [hunk 3]
```diff
@@ -193,34 +193,34 @@ SAVEFILE UPDATING/VERSIONING - 'Simplified', or rather, more coder-friendly ~Car
 		return 0
 	S.cd = "/"
 
-	S["version"] << SAVEFILE_VERSION_MAX		//updates (or failing that the sanity checks) will ensure data is not invalid at load. Assume up-to-date
+	WRITE_FILE(S["version"] , SAVEFILE_VERSION_MAX)		//updates (or failing that the sanity checks) will ensure data is not invalid at load. Assume up-to-date
```
Comment (optimumtact) [inline code/modules/client/preferences_savefile.dm]: does all the save file stuff still work fine like this?

**Comment by optimumtact** [comment]
assuming the save files work, this looks ready to go

File: code/modules/client/preferences_savefile.dm (hunk 3 above)
Comment (lzimann) [inline code/modules/client/preferences_savefile.dm]: Can't see why wouldn't.

Comment (Cyberboss) [inline code/modules/client/preferences_savefile.dm]: Yes

**Pull request merged**